src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -956,11 +978,22 @@ private static SegueJobService getSegueJobService(final PropertiesLoader propert
                 configuredScheduledJobs.add(syncMailjetUsers);
             }
 
+            if (eventPrePostEmailsEnabled) {
+                configuredScheduledJobs.add(eventReminderEmail);
+                configuredScheduledJobs.add(eventFeedbackEmail);
+            }
+
             segueJobService = new SegueJobService(configuredScheduledJobs, database);
 
             if (mailjetKey == null && mailjetSecret == null) {
                 segueJobService.removeScheduleJob(syncMailjetUsers);
             }
+
+            if (!eventPrePostEmailsEnabled) {
+                segueJobService.removeScheduleJob(eventReminderEmail);
+                segueJobService.removeScheduleJob(eventFeedbackEmail);
+            }
```

Comment:
If at some point down the line we want to stop it running, just removing the config property wouldn't work as it would still live in the qrtz_triggers / qrtz_jobs tables in the db, so it needs actually removing

Reply:
I suppose a comment before these removals wouldn't do any harm, to explain this and the other section like it for future reference!

Reply:
Thought it might be something like that 👍